### PR TITLE
MTU migration for AWS SDN

### PR DESCRIPTION
**Pull request by anuragthehatter**
This is for SDN. 
Same description as https://github.com/openshift/release/pull/32123

### CODE REVIEW

**Comment by anuragthehatter** [comment]
cc @openshift/team-sdn-qe

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml [hunk 1]
```diff
@@ -33,13 +33,13 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: installer-rehearse-aws
+- as: installer-rehearse-aws-mtu-migrate
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml:36]: Here do not need to change the job name, suggest to keep it unchanged.

Comment (anuragthehatter) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml]: Good idea. Thanks. Taken care of

Comment (jianlinliu) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml]: Once that, https://github.com/openshift/release/pull/32171/files#diff-99e029f94e9d5b7bbffc17380fbcf9794b70d267a2e9144171e2aac459a664b4 should have no change, forget to `make registry-metadata` in your local?

Comment (anuragthehatter) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml]: sorry, yep. Just pushed `make update`

**Comment by jianlinliu** [comment]
As usual, pls squash multiple commits into one.

**Pull request merged**